lib/common/ui/custom/dialog.dart

```diff
@@ -47,6 +47,110 @@ class CDialog extends StatefulWidget {
     ).show(context);
   }
 
+  static void showInternetUnavailableDialog(BuildContext context) {
```

Comment:
I confirmed I see this when I turn on Airplane mode:

![Screenshot_20240527-172916](https://github.com/getlantern/lantern-client/assets/6346213/c9014ea3-4fc6-46ac-91ff-84cebd4676a5)

Reply:
@atavism we are showing this dialog when there is no interconnection found and VPN switch disable.